pkgs/development/python-modules/libdecsync/default.nix

```
@@ -0,0 +1,18 @@
+{ lib, python3Packages }:
+
+python3Packages.buildPythonPackage rec {
+  pname = "libdecsync";
+  version = "1.7.1";
+
+  src = python3Packages.fetchPypi {
```

```suggestion
{ lib, buildPythonPackage, fetchPypi }:

buildPythonPackage rec {
  pname = "libdecsync";
  version = "1.7.1";

  src = fetchPypi {
```

Using python3Packages is not allowed in pythonPackages because it mixes python versions.